modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/main.es.js

```diff
@@ -260,14 +267,17 @@ class Form extends Component {
 			this._autoSave.dispose();
 		}
 
+		if (this._stateSyncronizer) {
```

Comment:
For some reason we weren't disposing this

Reply:
Changing from `disposeInternal` override to `disposed` in favor to use Metal's lifecycle. Debugging it, the problem was solved in this case.